ckanext/datawagovautheme/templates/home/snippets/search.html

```diff
@@ -15,7 +15,10 @@ <h3 class="heading">{{ _("Search data") }}</h3>
   <div class="tags">
     <h3>{{ _('Popular tags') }}</h3>
```

Comment:
And we should try to fix other occurrences of aggressive string truncation that make it harder for the user to understand what they're looking at.

e.g. From https://uat.landgate.links.com.au/dataset/general-clearance-areas-dmirs-022

![screen_shot_2019-02-18_at_12_58_51](https://user-images.githubusercontent.com/1672960/52928956-28cd9100-337d-11e9-98ac-c77eaaf81169.jpg)

is more readable if they're not truncated:
![screen shot 2019-02-18 at 13 00 58](https://user-images.githubusercontent.com/1672960/52928985-54507b80-337d-11e9-8cdc-e79414e6602e.png)

Reply:
We don't use breadcrumbs here so I didn't make any changes. I don't recommend it unless it is necessary.